### DISCUSSION TITLE
How to create a Welcome post?

### DISCUSSION THREAD

**Discussion by mzagorski** [dailydotdev/daily#1453] [General]
When I created the Squad, I deleted the Welcome post and now can't recreate it. Is there an option to restore or recreate it?

**Comment by idoshamun** [comment]
Hi @mzagorski,

We'll recreate it for you. Please share here the squad handle

**Comment by mzagorski** [comment]
Hi, 
The squad link: https://app.daily.dev/squads/snippetlibrary
Thanks

**Reply by idoshamun** [reply]
I fixed it :)
